src/main/java/org/folio/rmapi/RMAPIService.java

```diff
@@ -341,4 +355,22 @@ public Packages getPackages() {
       return packages;
     }
   }
+
+  public static class ResourceResult {
```

Comment:
I don't see why it has to be an inner class. To me it looks like a part of RM API Service model, like Title VendorId. Otherwise the methods look inconsistent, in some cases you return model classes in other - something else which belongs to Service

Reply:
Other model classes serve as mappings for RM API responses, ResourceResult and VendorResult are just wrapper classes that combine several RM API responses. Agree that they shouldn't be declared as inner classes, moved them to rmapi/result package